packages/nitro-protocol/tsconfig.json

```diff
@@ -13,6 +13,18 @@
     "resolveJsonModule": true,
     "downlevelIteration": true
   },
-  "include": ["src/**/*", "test/test-helpers.ts", "artifacts/**/*.json", "addresses.json"],
+  "include": [
+    "src/**/*",
+    "artifacts/contracts/Token.sol/Token.json",
+    "artifacts/contracts/NitroAdjudicator.sol/NitroAdjudicator.json",
+    "artifacts/contracts/test/TESTNitroAdjudicator.sol/TESTNitroAdjudicator.json",
+    "artifacts/contracts/CountingApp.sol/CountingApp.json",
+    "artifacts/contracts/examples/HashLockedSwap.sol/HashLockedSwap.json",
+    "artifacts/contracts/interfaces/IForceMoveApp.sol/IForceMoveApp.json",
+    "artifacts/contracts/ForceMove.sol/ForceMove.json",
```

Comment:
If in future we want to export another artifact not listed here, we will see a typescript error prompting us to include it.